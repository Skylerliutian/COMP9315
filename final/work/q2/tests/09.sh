./bnl 20 100 15
