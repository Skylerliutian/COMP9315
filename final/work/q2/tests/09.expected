Running: ./bnl 20 100 15

Frames:     [00] [01] [02] [03] [04] [05] [06] [07] [08] [09] [10] [11] [12] [13] [14]
Contents:    S96  S97  S92  S88  S93  R19  S89  S94  S98  S90  S95  S99  S91  S86  S87
PinCount:      0    0    0    0    0    0    0    0    0    0    0    0    0    0    0
Popularity:    2    2    1    0    1    1    0    1    2    0    1    2    1    0    0
Clock: 12

#requests: 2020
#releases: 2020
#hits    : 0
#reads   : 2020
